model/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, age)
+    super(age)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Same here, You need to inherit all parameters from the person class and initialize them, so please fix that.